.eslintrc

```diff
@@ -87,7 +87,7 @@
     /* Variables */
     "no-label-var": 1,
     "no-unused-vars": [
-      1,
+      "warn",
```

Comment:
There's a `--max-warnings` in gulpfile.js for the lint target that you also want to configure.

Reply:
Thanks! Forgot that